src/ck-ldb/ckgraph.h

```diff
@@ -63,10 +63,9 @@ class Edge {
     Edge(int _id, int _msgs, int _bytes) : id(_id), msgs(_msgs),
       bytes(_bytes) {
     }
-    ~Edge() { }
-    inline int getNeighborId() { return id; }
-    inline int getNumMsgs() { return msgs; }
-    inline int getNumBytes() { return bytes; }
+    inline int getNeighborId() const { return id; }
+    inline int getNumMsgs() const { return msgs; }
+    inline int getNumBytes() const { return bytes; }
     inline void setNumBytes(int _bytes) { bytes = _bytes; }
```

Comment:
No need for `inline` in these functions (same below).

Reply:
Fixed.